builtin_op_importers.cpp

```diff
@@ -1518,6 +1528,111 @@ DEFINE_BUILTIN_OP_IMPORTER(Size) {
   return {{weights}};
 }
 
+DEFINE_BUILTIN_OP_IMPORTER(Slice) {
+  ASSERT(inputs.size() == 1, ErrorCode::kINVALID_NODE);
+  nvinfer1::ITensor* tensor_ptr = &convertToTensor(inputs.at(0), ctx);
+  nvinfer1::Dims dims = tensor_ptr->getDimensions();
+  int nbDims = dims.nbDims;
```

Comment:
[Copying comment from #82]
This is used inconsistently below (sometimes it's used, and sometimes dims.nbDims is referenced directly).